components/Typography/src/UIFontDescriptor+MaterialTypography.m

```diff
@@ -36,24 +36,32 @@ + (nonnull UIFontDescriptor *)mdc_preferredFontDescriptorForMaterialTextStyle:
   // Store the system font family name to ensure that we load the system font.
   // If we do not explicitly include this UIFontDescriptorFamilyAttribute in the
   // FontDescriptor the OS will default to Helvetica.
-  static NSString *systemFontFamilyName;
+  static NSString *smallSystemFontFamilyName;
+  // For font size 20+, iOS returns a different font family. Query it separately.
+  static NSString *largeSystemFontFamilyName;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
-    UIFont *systemFont;
+    UIFont *smallSystemFont;
+    UIFont *largeSystemFont;
     if ([UIFont respondsToSelector:@selector(systemFontOfSize:weight:)]) {
-      systemFont = [UIFont systemFontOfSize:12 weight:UIFontWeightRegular];
+      smallSystemFont = [UIFont systemFontOfSize:12 weight:UIFontWeightRegular];
+      largeSystemFont = [UIFont systemFontOfSize:20 weight:UIFontWeightRegular];
     } else {
       // TODO: Remove this fallback once we are 8.2+
-      systemFont = [UIFont systemFontOfSize:12];
+      smallSystemFont = [UIFont systemFontOfSize:12];
+      largeSystemFont = [UIFont systemFontOfSize:20];
     }
-    systemFontFamilyName = [systemFont.familyName copy];
+    smallSystemFontFamilyName = [smallSystemFont.familyName copy];
+    largeSystemFontFamilyName = [largeSystemFont.familyName copy];
   });
 
   NSDictionary *traits = @{ UIFontWeightTrait : @(materialTraits.weight) };
+  NSString *fontFamily =
+      materialTraits.pointSize < 19.5 ? smallSystemFontFamilyName : largeSystemFontFamilyName;
```

Comment:
Nit/Suggestion: Should we make this 20 instead of 19.5? Apple's documentation says that "20 and above" should use SF Pro Display. Does the 19.5 look better visually as a breaking point?